src/simpleindex/__main__.py

```diff
@@ -4,33 +4,45 @@
 
 from starlette.applications import Starlette
 from starlette.requests import Request
-from starlette.responses import Response
+from starlette.responses import Response, StreamingResponse
 from starlette.routing import Route
 from uvicorn import run as run_uvicorn
 
 from . import configs, routes
 
 
-def _build_routes(key: str, route: routes.Route) -> typing.List[Route]:
-    async def page(request: Request):
-        response = await route.get_page(request.path_params)
-        return Response(
+def _to_http_response(
+    response: typing.Union[routes.Response, routes.StreamingResponse]
+):
+    # The order is crutial due the inheritance between *Response classes
+    if isinstance(response, routes.StreamingResponse):
```

Comment:
May be a good idea to move these logic into the response classes. This if-else is manually implemented polymorphism and a good refactoring candidate.

Reply:
It is a good idea. I am just wondering that if we put a `def to_http_response(self)` in the `*Response` class than the typing would stick to starlette. I do not think it is an issue, just want to raise it.

Reply:
I have pushed a commit with this change. Let me know what you think about it.